src/test/java/net/honux/chess/PawnTest.java

```diff
@@ -8,8 +8,11 @@
     @Test
     @DisplayName("흰색 폰이 생성되어야 한다")
     public void create() {
-        String color = "white";
+        verifyPawn("white");
+        verifyPawn("black");
+    }
 
+    private void verifyPawn(final String color) {
         Pawn pawn = new Pawn(color);
         assertThat(pawn.getColor()).isEqualTo(color);
     }
```

Comment:
중복제거를 해주신 점 좋습니다.
메서드 파라미터로 `final`을 주셨는데 이 부분은 어떤 이유에서 요렇게 작성하셨나요?

Reply:
이 부분은 별 생각없이 루카스에 있는 그대로 가져와서 사용했습니다.

이 [링크](https://stackoverflow.com/questions/266806/is-there-any-performance-reason-to-declare-method-parameters-final-in-java)에서 참고해서 제 생각을 말씀드리면,
파라미터나 지역 변수에 does not appear 한다는데, appear 하지 않는다는게 무슨 말인지 이해가 안가구요.
이 답변자에 따르면 주로 우리가 사용하는 목적에 부합해서만 사용해야 할거 같은 느낌이 드네요.

> There is a lot of argument over whether the final modifier on the method itself has any performance gain since the methods will be inlined by the optimizing compiler at runtime anyway,

부분을 보면 메서드에서 final을 쓰는건 많은 argument가 있다고하는 걸 봐선, 저는 사용하고 싶지않아졌습니다.

두번째 답변자의 답변은 다음과 같은데요.

> The only benefit to a final parameter is that it can be used in anonymous nested classes

anonymous nested classes 가 익명 중첩 클래스라고 하는데 제가 지금 단계에서 이 부분을 더 깊이 들어가야 할 건 아닌거같아서 여기까지만 공부햇습니다.

Reply:
이는 성능보다는 코드 품질과 관계있습니다. Kyu가 찾으신 질문은 성능과 관련된 문제여서 왜 써야하는지를 명확히 드러내주지 않습니다.
메서드 파라미터가 final이면 코드블럭에서 변경되지 않음을 명시적으로 보여줍니다.
final 키워드 자체에 대해서 더 학습해보세요!